src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Actions/Row/submit.html.twig

```diff
@@ -39,7 +39,7 @@
 {% set extra_route_params = action.options.extra_route_params %}
 
 {% for name, field in extra_route_params %}
-  {% set route_params = route_params|merge({ (name) : (record[field]) }) %}
+  {% set route_params = route_params|merge({ (name) : (record[field] ?? field) }) %}
```

Comment:
It reverts a regression no? Because in 177 there is already this `field` fallback

Reply:
@sowbiba any update on that?

Reply:
@jolelievre we discussed about this. the fallback you're talking about is on link.html.twig file.
Are you ok with that ?

Reply:
It's useful indeed, but we might have to deal with a conflict someday In case the route parameter name matches a record field name.. Do we accept it for now and think about a fix later or should we anticipate it?
ex:
``````

Reply:
ex:

Reply:
ex:

```php
$record = [
  'id' => 15,
  'name' => 'toto',
  'action' => 'deliver',
];
```

```yaml
route:
   some_route:
       url: /my_route/{action}
`
``

If we want to specify the route with a specific parameter `action` it will use the record field, not the option from the row action